src/cloudant/document.py

```diff
@@ -65,6 +65,21 @@ def __init__(self, database, document_id=None):
             self['_id'] = self._document_id
         self.encoder = self._client.encoder
 
+    def copy(self):
+        """
+        Create a copy of the Document, including any locally cached content.
+        """
+        return self.__copy__()
+
+    def __copy__(self):
+        """
+        Copy the document, for use in copy.copy() calls. See `copy()`.
+        """
+        cpy = Document(self._database, document_id=self._document_id)
+        for k, v in self.items():
+            cpy[k] = v
```

Comment:
Could `v` be a mutable type here? If so it'll get passed by reference and you won't get a deep copy.

Reply:
I don't intend for it to be a deep copy, only a shallow copy. See "What", "Added support for (shallow) copying Document objects." :) 

Whether this is the right intent we can discuss, but I took the normal dict copy behaviour.